packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -36,14 +40,15 @@ class _AccountPictures extends StatelessWidget {
             children: (otherAccountsPictures ?? <Widget>[]).take(3).map<Widget>((Widget picture) {
               return Padding(
                 padding: const EdgeInsetsDirectional.only(start: 8.0),
-                child: Semantics(
-                  container: true,
-                  child: Container(
+                child: Padding(
```

Comment:
Could these two Padding widgets be combined?

Reply:
We could write `const EdgeInsets.only(left: 8.0, bottom: 8.0).add(const EdgeInsetsDirectional.only(start: 8.0))`

But I feel like this would be a bit too unreadable.

Are we sure that we can not merge both of them into one single EdgeInsetsDirectional? I am not aware of the design guidelines for this widget. But if yes we could end up with just `const EdgeInsetsDirectional.only(start: 16.0, end: 8.0)`

Reply:
Oh I see, I missed that it was using both `left` and `start`.  I think it's alright to keep this as-is with two Padding widgets for now, since that's effectively how it was before this PR.